pypeit/spectrographs/gemini_flamingos.py

```diff
@@ -156,6 +157,12 @@ def default_pypeit_par(self):
         # Do not correct for flexure
         par['flexure']['spec_method'] = 'skip'
 
+        # Sensitivity function parameters
+        par['sensfunc']['algorithm'] = 'IR'
+        par['sensfunc']['polyorder'] = 8
+        # place holder for telgrid file
+        par['sensfunc']['IR']['telgridfile'] = resource_filename('pypeit', '/data/telluric/TelFit_LasCampanas_3100_26100_R20000.fits')
```

Comment:
We don't have a file for CTIO right? I guess they are close enough together.

Reply:
Not yet. Add a ToDo